### PR TITLE
COMMERCE-7821 Fixed option value localization

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-7821

Re-sent from https://github.com/liferay-commerce/liferay-portal/pull/3512

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
@jaydawu can it be forwarded?

**Comment by jaydawu** [comment]
Hi @alessiorendina. This is under QA review. If everything is good, it will be forwarded.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 1]
```diff
@@ -363,6 +363,55 @@ definition {
...
+
+		task ("And change the master catalog language as new") {
+			CommerceCatalog.openCatalogsAdmin();
+
+			CommerceCatalog.editCatalogEntry(
+				catalogLanguage = "English (United Kingdom)",
+				catalogName = "Master");
+		}
+
+		task ("And add a option within a new value") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:387]:
```suggestion
		task ("And an option with an option value is added") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 2]
```diff
@@ -363,6 +363,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-7821. The information under options should be supported by localization"
+	@priority = 5
+	test CanBeLocalizedWithOptionsInformation {
+		property portal.acceptance = "true";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:369]: @jaydawu this test should not be part of the relevant, please change the property and lower the test priority to 3. Thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
@lorenzolapescara update the test already. Please review. Thanks.

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
@lorenzolapescara update the test already. Please review. Thanks.

**Review by lorenzolapescara — APPROVED** [review]

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**